recipes/modern-cpp-kafka/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+set(CMAKE_CXX_STANDARD 17)
+
+find_package(RdKafka REQUIRED CONFIG)
+find_package(ModernCppKafka REQUIRED CONFIG)
+
+add_executable(${PROJECT_NAME} test_package.cpp)
+target_link_libraries(${PROJECT_NAME} RdKafka::rdkafka ModernCppKafka::ModernCppKafka)
```

```suggestion
target_link_libraries(${PROJECT_NAME} ModernCppKafka::ModernCppKafka)
```

This should arrive as a transitive dependency